crime-application-tracking/src/main/java/uk/gov/justice/laa/crime/application/tracking/service/EformsDecisionHistoryService.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.justice.laa.crime.application.tracking.service;
+
+import io.micrometer.observation.Observation;
+import io.micrometer.observation.ObservationRegistry;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.laa.crime.application.tracking.client.MaatCourtDataApiClient;
+import uk.gov.justice.laa.crime.application.tracking.model.EformsDecisionHistory;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class EformsDecisionHistoryService {
+    private static final String SERVICE_NAME = "eformsDecisionHistoryService";
+    public static final String WROTE_TO_RESULTS = "Y";
```

Comment:
`WROTE_TO_RESULTS` can probably be private

Reply:
Done